addons/godot_xml/xml.gd

```diff
@@ -194,6 +199,11 @@ static func _attach_node_data(node: XMLNode, parser: XMLParser) -> void:
         # we therefore strip "blankets", resulting in only actual content slipping into .content
         node.content = parser.get_node_data().strip_edges().lstrip(" ").rstrip(" ")
 
+static func _attach_node_cdata(node: XMLNode, parser: XMLParser) -> void:
+    # WARNING: The current implementation of XMLParser.get_node_name() does not raise an error for nodes of type NODE_CDATA.
+    # Instead, it returns the CDATA content. This behavior is undocumented and may change in future versions of Godot.
+    # It is recommended to open an issue on the Godot repository to address this inconsistency.
```

Comment:
I opened an issue: godotengine/godot#89120. We'll need to wait until it is resolved.

Reply:
The behaviour is confirmed to be correct, a PR will land into Godot 4.3. I removed this note